### ISSUE TITLE
Flux fill - train - triplets and pairs

### ISSUE THREAD

**Issue by duchamps0305** [lrzjason/T2ITrainer#18]
Thanks for your tool, it’s a great alternative to onetrainer that also offers flux fill training. 

I saw that your example trains on triplets. I would however want to train on just image pairs (image+mask) for an inpainting task. 
I saw that your example is about object removal, I want to instead do an object lora on a pair of sunglasses. 

Is this something possible? 

maybe modifying something like this? I'm not code-abled so this is just an idea...

```
# Load image pairs
original_image = Image.open(image_path)
mask_image = Image.open(mask_path)

# ... preprocessing steps ...

# Pass the original image and mask to the model
loss = model(original_image, mask_image, timestep)

# ... backpropagation and optimization steps ...
```

**Comment by lrzjason** [comment]
The easiest way is to copy you original image and rename with suffix _G. It doesn't require to change any code

**Comment by duchamps0305** [comment]
you are amazing. thank you so much.
